docs/submit/osg-flock.md

```diff
@@ -87,79 +99,96 @@ Start by setting up the OSG YUM repositories following the
 versions will not work.
 
 Once the YUM repositories are setup, install the `osg-flock` convenience RPM that installs all
-required packages:
+required packages. Example on a RHEL 7 host:
 
 ```console
-root@host # yum install osg-flock
+# yum install https://repo.opensciencegrid.org/osg/3.6/osg-3.6-el7-release-latest.rpm
+# yum install osg-flock
 ```
 
+### Upgrading
+
+Upgrading from previous versions should be as simple as switching to OSG 3.6, and then
+issuing `yum upgrade`. If you made local config changes, please verify that the files under
+`/etc/condor/config.d` were renamed/disabled during the upgrade.
+
+Note that in some older versions of the package, the Gratia config was kept in 
+`/etc/gratia/condor/ProbeConfig`. The new location is `/etc/gratia/condor-ap/ProbeConfig`.
+
+The Open Science Pool will no longer accept GSI authentcation. Access points still configured
+with GSI, will have to be upgraded to OSG 3.6 and switched over to token authentication as
+described in this document.
 
 Configuring Reporting via Gratia
 --------------------------------
 Reporting to the OSG accounting system is done using the _Gratia_ service, which consists of multiple _probes_.
-HTCondor uses the "condor" probe, which is configured in `/etc/gratia/condor/ProbeConfig`;
+HTCondor uses the "condor" probe, which is configured in `/etc/gratia/condor-ap/ProbeConfig`;
 we provide a recommended configuration for flocking.
 
-1. Copy over the recommended probe configuration:
-
-        :::console
-        root@host # cp /etc/gratia/condor/ProbeConfig-flocking /etc/gratia/condor/ProbeConfig
-
-1. Fill in the value for `ProbeName` with the hostname of your access point, with the following format:
-
-        :::xml
-        ProbeName="condor:<HOSTNAME>"
-        
 1. Fill in the value for `SiteName` with the Resource Name you registered in Topology (see
    [instructions above](#register-your-access-point-in-osg-topology)).
    For example:
    
         :::xml
         SiteName="OSG_US_EXAMPLE_SUBMIT"
 
-1. Enable the Gratia Probe.
+1. Enable the Gratia Probe:
 
         :::xml
         EnableProbe=1
 
-1. Under the section `Title2` make sure to set the following (if not already there):
-  
+1. Add a section to map unknown records to OSG:
+        
         :::xml
-        MapUnknownToGroup="1"
-        MapGroupToRole="1"
         VOOverride="OSG"
+        MapGroupToRole="1"
+        MapUnknownToGroup="1"
```

Comment:
This should be automatically handled by the packaging now: https://github.com/opensciencegrid/osg-flock/blob/master/rpm/osg-flock.spec#L36-L52

Reply:
Ah, fixed.